### PR TITLE
feat: Add plot function

### PR DESCRIPTION
**Pull request by coolalexzb**
In this PR, I aim to add `plot` function to our project using Matplotlib. Now, users can get only CPU utilization graph after they run the script. I will add other metrics in further steps. From [NVIDIA System Management Interface](https://developer.nvidia.com/nvidia-system-management-interface), most of the metrics are useful, which means that most of the metrics plotted by `wandb` are useful.

### CODE REVIEW

**Comment by coolalexzb** [comment]
One python usage question: I want to use filter function `list_of_dirs = list(filter(lambda p: p.is_dir(), list_of_paths))` to get all the directories under wandb directory, but get an empty list when using filter function, but I can get right result if I use for loop and check.

**Comment by coolalexzb** [comment]
After users run `run.py` script, they will get 3 figures (if GPU used else 2 figures) of performance of CPU and GPU. Following is the result of running `1Lbb-likelihoods-hepdata` 
屏幕快照 2020-06-26 下午4 08 39

**Comment by coolalexzb** [comment]
Is the solution to resolve conflicts the same as we used before?
```
git fetch --all
git rebase upstream/master # Resolve any merge conflicts during the rebase
git push --force-with-lease
```

**Comment by lukasheinrich** [comment]
@coolalexzb should the plots show some markers or are they intended to be empty (and just show the structure?)

**Comment by coolalexzb** [comment]
@lukasheinrich  The plots showed above just have one dot on each graph because it only takes about 30 seconds to run` 1Lbb-likelihoods-hepdata` and only one status is recorded.

**Comment by matthewfeickert** [comment]
> @lukasheinrich The plots showed above just have one dot on each graph because it only takes about 30 seconds to run` 1Lbb-likelihoods-hepdata` and only one status is recorded.

As we already discussed you're going to need to use other models that take longer, or create unrealistic models that have many bins in them to have things that take more than one iteration to run. It would be good to see what the plots actually look like.

**Comment by coolalexzb** [comment]
@matthewfeickert How can I create a larger models that will run for a longer time? Is there any existing code snippets to use? Or Is there any approach to make use of `1Lbb-likelihoods-hepdata` since currently I just choose model = [750, 100]. I tried to use the entire hepdata by making model parameters empty but a bug occurs. 
屏幕快照 2020-06-30 下午6 02 59

**Comment by coolalexzb** [comment]
New Pr aims to save all the graph as png locally

File: scripts/plot.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+def load(directory_name):
+    directory, path = find_latest(directory_name)
+    output_dic = {}
+
+    clock = 0
+    while not path.exists():
+        clock += 1
+        time.sleep(1)
+        if clock >= 60:
+            raise FileExistsError(f"{path} is not found!")
```
Comment (matthewfeickert) [inline scripts/plot.py:29]: What dos this clock section do? It isn't clear how the logic in this section is going to do anything other then wait 60 seconds before raising an error.

**Pull request merged**